cibuildwheel/util.py

```diff
@@ -1,5 +1,11 @@
 from fnmatch import fnmatch
 
+filter_wheels_cmd = '''\
+import sys, glob, itertools, pip; \
+sys.stdout.write(\
+' '.join(f for f in itertools.chain(*map(glob.iglob, sys.argv[1:])) \
+if pip.wheel.Wheel(f).supported()))\
+'''
```

Comment:
I'm finding this quite hard to understand. What does this do, and might there be a more elegant way to do it? I'm not averse to separate code for each platform by the way!